libmcount/internal.h

```diff
@@ -446,6 +457,8 @@ struct mcount_disasm_info {
 	int			copy_size;
 	bool			modified;
 	bool			has_jump;
+	uint8_t			nr_branch;
+	struct			cond_branch_info branch_info[MAX_COND_BRANCH];
```

Comment:
No struct name should go left, please adjust the indentation if needed.

Reply:
I usually use spaces for structs. I'm not sure about the indentation with tabs. I hope I got it right this time =)